tsconfig.json

```diff
@@ -24,7 +24,7 @@
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx",
-    "*.d.ts",
+    "*.d.ts", "models/leaderboardSchema.tsx", "models/leaderboardSchema.ts", "pages/api/leaderboardAPI/leaderboardAPI.tsx",
```

Comment:
Line 25 and 26 in this file already covers all `.tsx` and `.ts` files

Reply:
Don't know why that is added, will remove